Gax.sln

```diff
@@ -48,6 +48,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "MSBuild", "MSBuild", "{68EA
 		ReleaseVersion.xml = ReleaseVersion.xml
 	EndProjectSection
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Google.Apis.Auth", "..\google-api-dotnet-client\Src\Support\Google.Apis.Auth\Google.Apis.Auth.csproj", "{8F05BBAB-7B26-44DF-A139-4B631A5C6A85}"
```

Comment:
Google.Apis.Auth has now been released, so you should be able to revert this and use the new package.

Reply:
Yep, will do